alpine/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM alpine
+RUN apk --no-cache add curl python
```

Comment:
can you avoid installing `curl`? it looks like `wget` exists on alpine. this will help keeping image smaller.